doc/getting-started/acrn_quick_setup.sh

```diff
@@ -72,7 +72,7 @@ function upgrade_sos()
         echo "Clear Linux version $sos_ver is already installed. Continuing to setup SOS..."
     else
         echo "Upgrading Clear Linux version from $VERSION_ID to $sos_ver ..."
-        swupd verify --fix --picky -m $sos_ver 2>/dev/null
+        swupd diagnose --fix --picky --version $sos_ver 2>/dev/null
```

Comment:
Can/Should this check that the Clear Linux version supports this swupd command?  There should be error detection in this script to catch if the ``swupd diagnose`` command fails, right?  Similar comment for the changes below that depend on a newer version of Clear Linux to succeed.